### DISCUSSION TITLE
Token limit? ~3500?

### DISCUSSION THREAD

**Discussion by 0xAskar** [llm-workflow-engine/llm-workflow-engine#45] [Q&A]
What is the token limit? I was thinking it was 4097 as suggested online, but not sure what the limit is here. Currently, through error and trial, I'm hitting a hard limit at approx. ~3500 tokens. I'm using OpenAI's [tokenizer](https://beta.openai.com/tokenizer) to calculate. Is 3500 correct?

**Comment by thehunmonkgroup** [comment]
Browser backend operates same as chat.openai.com, API backend is 4096-ish, I think there's a bit of overhead in the request text itself.
